samples/nrf9160/mqtt_multi_service/prj.conf

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+
+# Location Services nonsense
+CONFIG_LOCATION=y
+CONFIG_NRF_MODEM_LIB=y
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LOCATION_METHOD_GNSS=y
+CONFIG_LOCATION_METHOD_CELLULAR=y
+CONFIG_NRF_CLOUD_AGPS=y
+CONFIG_NRF_CLOUD_PGPS=y
+CONFIG_MULTICELL_LOCATION_SERVICE_NRF_CLOUD=y
+CONFIG_MODEM_INFO_ADD_NETWORK=y
+CONFIG_MODEM_INFO=y
+CONFIG_MODEM_INFO_ADD_NETWORK=y
+CONFIG_MODEM_JWT=y
```

Comment:
Ah, these are required for the PGPS stuff, it's using REST apparently

That wasn't very clear from how disorderly my prj.conf was, though -- I have since reorganized it!

Reply:
No, it should not be.  The location library should support MQTT for everything it needs.